### ISSUE TITLE
stalled graph tree locks

### ISSUE THREAD

**Issue by chilek** [Cacti/cacti#3489]
Hello,

is there a way to force cacti to auto-unlock graph trees after some user inactivity timer expires (I'd prefer it to be per user configurable, but global setting would be great too)?

**Comment by TheWitness** [comment]
Interesting feature request.

**Comment by chilek** [comment]
@TheWitness by the way - does cacti participate in github sponsor program? Would be very nice to give some tip monthly for you!

**Comment by TheWitness** [comment]
Does that exist?  @netniV & @ronytomen tip jar?  Let's make it happen!

**Comment by netniV** [comment]
I don't believe that Cacti itself has signed up to the sponsor program, though I have had a coffee donated to me through it.  I'll speak with @ronytomen as I need to catch up on some other org level stuff and see what we are going to do.

**Comment by netniV** [comment]
On the auto unlock feature, wasn't this added recently?

**Comment by TheWitness** [comment]
No, the feature was to allow an admin to on mass unlock.

**Comment by netniV** [comment]
Ah yea, I knew it was similar.
